src/Symfony/Component/String/Tests/Slugger/AsciiSluggerTest.php

```diff
@@ -44,4 +44,27 @@ public function testSlug(string $expected, string $string, string $separator = '
 
         $this->assertSame($expected, (string) $slugger->slug($string, $separator, $locale));
     }
+
+    public function provideSlugEmojiTests(): iterable
+    {
+        yield [
+            'un-chat-qui-sourit-chat-noir-et-un-tete-de-lion-vont-au-parc-national',
+            'un 😺, 🐈‍⬛, et un 🦁 vont au 🏞️',
```

Comment:
PR desc need to be updated, the « fr » seems wrong

Reply:
What does seems wrong ?

Reply:
et a 🦁 vont au
VS
et un 🦁 vont au

Reply:
Oh, the **description**, got it.